### DISCUSSION TITLE
How to run prefect agent in background in centos

### DISCUSSION THREAD

**Discussion by arunajakhmola** [PrefectHQ/prefect#4962] [Q&A] [answered]
I want to run prefect agent as background process. prefect agent local start

also need to check the status of the running agent (health check) for health check up i used below command

prefect agent local start -f --agent-address http://100.64.114.291:8081

but output of this command is always blank curl --fail --silent "http://100.64.114.291/api/health"

Please help me .

**Comment by zanieb** [comment, marked as answer]
You can use `prefect agent local install` with `supervisor` http://supervisord.org/ or use GNU `screen` (e.g. `screen -S`)

If you can successfully ping the health endpoint, it is healthy.
```
❯ curl "http://localhost:8081/api/health"
{}%                                                                                                                  
```
is expected behavior

**Reply by arunajakhmola** [reply]
Hello @madkinsz  Thank you for your response. 
output of  curl "http://100.64.114.291:8081/api/health" for me is {}  not {}%.
only {} in response is the expected format ???

**Reply by zanieb** [reply]
It's an empty JSON blob. `{}` is correct. The `%` is from my terminal indicating a line without a newline at the end.

**Reply by arunajakhmola** [reply]
Thank you :)
